http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/resources/HttpVersionBaseResource.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.ts.http.restclient.reactive.resources;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.Response;
+
+import io.smallrye.mutiny.Uni;
+
+@ApplicationScoped
+@Path("/httpVersion")
+public class HttpVersionBaseResource {
+
+    @GET
+    @Path("synchronous")
+    public Response sayHello(@QueryParam("name") String name) {
+        return Response.ok("Hello " + name).build();
```

Comment:
Hello is fine, you can keep it if you want as it tests query param. It would be nice to have incoming request protocol in response. This way, you can check that REST client is telling you a true about used HTTP version. I propose to inject `RoutingContext`, then `io.vertx.ext.web.RoutingContext#request` -> `io.vertx.core.http.HttpServerRequest#version` should always return `HTTP_2`.